packages/client/hmi-client/src/components/workflow/scenario-templates/value-of-information/value-of-information-scenario.ts

```diff
@@ -0,0 +1,317 @@
+import _ from 'lodash';
+import { BaseScenario } from '@/components/workflow/scenario-templates/base-scenario';
+import * as workflowService from '@/services/workflow';
+import { operation as ModelOp } from '@/components/workflow/ops/model/mod';
+import { operation as ModelConfigOp } from '@/components/workflow/ops/model-config/mod';
+import { operation as SimulateCiemssOp } from '@/components/workflow/ops/simulate-ciemss/mod';
+import { operation as CompareDatasetOp } from '@/components/workflow/ops/compare-datasets/mod';
+import { operation as InterventionOp } from '@/components/workflow/ops/intervention-policy/mod';
+import { OperatorNodeSize } from '@/services/workflow';
+import {
+	createModelConfiguration,
+	getModelConfigurationById,
+	setParameterDistribution
+} from '@/services/model-configurations';
+import { ChartSetting, ChartSettingType } from '@/types/common';
+import { updateChartSettingsBySelectedVariables } from '@/services/chart-settings';
+import { AssetType, ParameterSemantic } from '@/types/Types';
+import { getInterventionPolicyById } from '@/services/intervention-policy';
+import { useProjects } from '@/composables/project';
+import { formatUniformDistribution } from '../scenario-template-utils';
+
+/*
+ * Value of information scenario
+
+	Configure the model to represent the extremes of uncertainty for some
+	parameters, then simulate into the near future with different intervention
+	policies and compare the outcomes.
+
+  Users can input a model, a model configuration, a set of interventions,
+  and a set of parameters. For each parameter, a min and max value will be specified,
+  resulting in the creation of new model configurations for each parameter with that
+	parameter set to a uniform.
+
+	Example:
+	2 interventions, 2 parameters
+  Model Node
+  |
+  +-- Model Config Node (Uniform distrubiton for Param 1)
+  |     |
+  |     +-- Simulate Node (Intervention 1)
+  |
+  +-- Model Config Node (Uniform distrubiton for Param 2)
+  |     |
+  |     +-- Simulate Node (Intervention 1)
+  |
+  +-- Model Config Node (Uniform distrubiton for Param 1)
+  |     |
+  |     +-- Simulate Node (Intervention 2)
+  |
+  +-- Model Config Node (Uniform distrubiton for Param 2)
+        |
+        +-- Simulate Node (Intervention 2)
+ */
+export class ValueOfInformationScenario extends BaseScenario {
+	public static templateId = 'value-of-information-scenario';
+
+	public static templateName = 'Value of information';
+
+	modelSpec: { id: string };
+
+	modelConfigSpec: { id: string };
+
+	interventionSpecs: { id: string }[];
+
+	simulateSpec: { ids: string[] };
```

Comment:
I didn't notice this before, any particular why the two different ways of plurals `interventionSpecs: { id: string }[];` and `simulateSpec: { ids: string[] };` ? 

Eg why have both `{ ids: string[] }` and `{ id: string }[]` ?

Reply:
For the `simulateSpecs` I just wanted to hold a list of states/observables that the user wants to see there which can represent by an array of ids.  For the interventionSpecs there were some talks about potentially renaming/making new intervention policies within the template, so I wanted to keep a list of objects just in case we needed to keep more than just an `id` when/if that happens